src/App.vue

```diff
@@ -80,17 +83,44 @@ export default {
   },
 
   computed: {
-    authAPIURL () {
+    /** The Auth API string. */
+    authApiUrl (): string {
       return sessionStorage.getItem('AUTH_API_URL')
     },
 
-    showLoadingContainer () {
-      return !this.dataLoaded && !this.dashboardUnavailableDialog && !this.accountAuthorizationDialog
+    /** The Business ID string. */
+    businessId (): string {
+      return sessionStorage.getItem('BUSINESS_ID')
+    },
+
+    /** Is True if loading container should be shown, else False. */
+    showLoadingContainer (): boolean {
+      return (!this.dataLoaded && !this.dashboardUnavailableDialog && !this.accountAuthorizationDialog)
+    },
+
+    /** Is True if route is Signin, else False. */
+    isSigninRoute (): boolean {
+      return Boolean(this.$route.name === 'signin')
```

Comment:
Is this used elsewhere as well? Potential constant or if there is multiple route names, perhaps an enum?

Reply:
It's used in 3 or 4 places. I'll change it to a constant in my next PR.